rules/robust.yml

```diff
@@ -0,0 +1,172 @@
+groups:
+- name: AnomalyRobust
+  interval: 1m
+  rules:
+
+    ################################################################################
+    #                     Constants                                                #
+    ################################################################################
+
+    - record: anomaly:robust:mad_multiplier
+      expr: |-
+        2
+
+    - record: anomaly:robust:margin_multiplier
+      expr: |-
+        0.5
+
+    - record: anomaly:robust:sparse_threshold
+      expr: |-
+        5/60
+
+    ################################################################################
+    #                     Base Rules                                               #
+    ################################################################################
+
+    - record: anomaly:robust:select
```

Comment:
Suggestion: If we call the existing approach `adaptive`, how about calling this approach `stable` since it builds bands that are stable and more resistant to outliers?